### ISSUE TITLE
Cannot register user for multiple Applications using GO-client Library.

### ISSUE THREAD

**Issue by arihant30** [FusionAuth/fusionauth-issues#1576]
### Description 
I am trying to use the go-client library to integrate fusionauth. Comments on the github repo (https://github.com/FusionAuth/go-client/blob/master/pkg/fusionauth/Client.go#L2761) say that if I only pass the UserRegistration object, the user will be registered for the application whose ID is provided.
But when I am trying to implement the same workflow, I am getting the following error:

"You must specify either the [user.email] or [user.username] property. If you are emailing the user you must specify the [user.email]. user.password: You must specify the [user.password] property."

This is how I am initializing the request for the API:

fusionauthrequest := fusionauth.RegistrationRequest{
Registration: fusionauth.UserRegistration{
ApplicationId: applicationId,
Username: request.Email,
},
}

### Affects versions
1.32.1

### Expected behavior
Expect fusionauth to add registration for existing user.

**Comment by mooreds** [comment]
I think this is a duplicate of https://github.com/FusionAuth/go-client/issues/48 

@arihant30 there is a workaround mentioned in that issue.

**Comment by arihant30** [comment]
> I think this is a duplicate of https://github.com/FusionAuth/go-client/issues/48 
> 
> @arihant30 there is a workaround mentioned in that issue.

Yes I found the post later and managed to follow the workaround mentioned. 

Another problem which I've realized is related to this is: since we cannot send the User object if we want to register an existing user for a different application, we can't have a user set different passwords for different applications. Is there a way around that?

**Comment by mooreds** [comment]
> Another problem which I've realized is related to this is: since we cannot send the User object if we want to register an existing user for a different application, we can't have a user set different passwords for different applications. Is there a way around that?

@arihant30 that seems more like a support question. The right place for support questions:

* if you have a paid edition which includes support, please open a support ticket: https://account.fusionauth.io/account/support/
* if you do not, please post in the forum for community support: https://fusionauth.io/community/forum/

Thanks!
